File: 2019/J1.cpp
/**
 *    author:  sjay05
**/
#include <bits/stdc++.h>

using namespace std;

int main() {
    int a1, b1, a2, b2, a3, b3;
    scanf("%d", &a3);
    scanf("%d", &a2);
    scanf("%d", &a1);
    scanf("%d", &b3);
    scanf("%d", &b2);
    scanf("%d", &b1);
    int a_total, b_total;
    a_total = a3*3 + a2*2 + a1;
    b_total = b3*3 + b2*2 + b1;
    if (a_total == b_total) printf("T\n");
    if (a_total > b_total) printf("A\n");
    if (a_total < b_total) printf("B\n");
}

File: 2017/J1.py
"""
author: sjay05
"""
x = input()
y = input()

def find_quadrant(x, y):
    if x > 0 and y > 0:
        return "1"
    elif x < 0 and y > 0:
        return "2"
    elif x > 0 and y > 0:
        return "3"
    else:
        return "4"

print find_quadrant(x, y)
